pages/docs/reference/whatsnew12.md

```diff
@@ -0,0 +1,458 @@
+* A regular module that targets a specific platform and can either be a dependency of platform modules or depend on platform modules.
+
+When you compile a multiplatform project for a specific platform, the code for both the common and platform-specific parts is generated.
+
+A key feature of the multiplatform project support is the possibility to express dependencies of common code on platform-specific parts through **expected and actual declarations**. An expected declaration specifies an API (class, interface, annotation, top-level declaration etc.). An actual declaration is either a platform-dependent implementation of the API or a typealias referring to an existing implementation of the API in an external library. Here's an example:
+
+In common code:
+
+```kotlin
+// expected platform-specific API:
+expect fun hello(world: String): String
+
+fun greet() {
+    // usage of the expected API:
+    val greeting = hello("multi-platform world")
+    println(greeting)
+}
+
+expect class URL(spec: String) {
+    open fun getHost(): String
+    open fun getPath(): String
+}
+```
+
+In JVM platform code:
+
+```kotlin
+actual fun hello(world: String): String =
+    "Hello, $world, on the JVM platform!"
+
+// using existing platform-specific implementation:
+actual typealias URL = java.net.URL
+```
+
+See the [documentation](http://kotlinlang.org/docs/reference/multiplatform.html) for details and steps to build a 
+multiplatform project.
+
+### Writing multiplatform unit tests
+
+It is also possible to write tests in a common project so that they will be compiled and run in each platform project. 
+There are 4 annotations provided in `kotlin.test` package to markup tests in common code: `@Test`, `@Ignore`, 
+`@BeforeTest` and `@AfterTest`.
+
+In JVM platform these annotations are mapped to the corresponding JUnit 4 annotations and in JS they are already 
+available since 1.1.4 to support JS unit testing.
+
+In order to use them you need to add a dependency on `kotlin-test-annotations-common` to your common module, 
+on `kotlin-test-junit` to your JVM module, and on `kotlin-test-js` to the JS module.
+
+## Other Language Features
+
+### Array literals in annotations
+
+Starting with Kotlin 1.2, array arguments for annotations can be passed with the new array literal syntax instead 
+of the `arrayOf` function:
+
+```kotlin
+@CacheConfig(cacheNames = ["books", "default"])
+public class BookRepositoryImpl {
+    // ...
+}
+```
+
+The array literal syntax is constrained to annotation arguments.
+
+### Lateinit top-level properties and local variables
+
+The `lateinit` modifier can now be used on top-level properties and local variables. The latter can be used, 
+for example, when a lambda passed as a constructor argument to one object refers to another object 
+which has to be defined later:
+
+<div class="sample" markdown="1" data-min-compiler-version="1.1">
```

Comment:
Shouldn't `data-min-compiler-version` be 1.2 everywhere?

Reply:
Fixed.